src/server/items/controllers.js

```diff
@@ -30,5 +30,32 @@ export default {
       }
       cb();
     });
+  },
+
+  removeAll: (req, res, cb) => {
+    const errorList = [];
+    db.createReadStream({
+      start: 'item1',
+      end: 'item' + '\xFF'
```

Comment:
We are using 'itemIncrement' to count number of items in DB.
It will remove itemIncrement variable too.